trench/models.py

```diff
@@ -61,7 +62,7 @@ def primary_exists(self, user_id: Any) -> bool:
 
 class MFAMethod(Model):
     _BACKUP_CODES_DELIMITER = "|"
-
+    id = AutoField(primary_key=True)
```

Comment:
Any reason for the switch from `BigAutoField` to normal sized `AutoField`?

On positioning here, this is fine since there is a single model, otherwise it might be preferable to specify this in `apps.py`

ex.

```python
class TrenchConfig(AppConfig):
    default_auto_field = "django.db.models.BigAutoField"
    name = "trench"
    verbose_name = "django-trench"